buildenv/java/BUILD

```diff
@@ -12,7 +12,10 @@ package(default_visibility = ["//visibility:public"])
     default_java_toolchain(
         name = "toolchain_jdk_{version}".format(version = version),
```

Comment:
because this name is not unique by platform we can't call this multiple times for each platform. I wasn't sure what changing the name of this would entail, so using a platform-controlled config setting seemed like decent workaround